frontend/pages/admin/pacta-version/[id].vue

```diff
@@ -0,0 +1,110 @@
+<script setup lang="ts">
+import { type PactaVersion, type PactaVersionChanges } from '@/openapi/generated/pacta'
+
+const router = useRouter()
+const { pactaClient } = useAPI()
+const { error: { withLoadingAndErrorHandling, handleOAPIError } } = useModal()
+const { fromParams } = useURLParams()
+
+const id = presentOrCheckURL(fromParams('id'))
+
+const prefix = 'admin/pacta-version/[id]'
+const persistedPactaVersion = useState<PactaVersion>('totally different???')
```

Comment:
Probably not the state name we want

Reply:
Lol thanks. I had a very very odd bug where these kept going back into sync.